src/rebar3_ex_doc.erl

```diff
@@ -554,3 +561,34 @@ mermaid_add(Vsn) when is_list(Vsn) ->
     }}];
 mermaid_add(Vsn) ->
     ?RAISE({mermaid_vsn_not_string, Vsn}).
+
+merge_before_closing_body_tag(MermaidMap, Custom) ->
+    try
+        do_merge_before_closing_body_tag(MermaidMap, Custom)
+    catch
+        _:_:_ ->
+            ?RAISE({invalid_before_closing_body_tag, Custom})
+    end.
+
+do_merge_before_closing_body_tag(undefined, Custom) ->
+    Custom;
+do_merge_before_closing_body_tag(MermaidMap, undefined) ->
+    MermaidMap;
+do_merge_before_closing_body_tag(MermaidMap, {M,F,Args}) 
+        when is_atom(M) and is_atom(F) and is_list(Args) ->
+    CustomMap = #{html => erlang:apply(M, F, [html|Args]),
```

Comment:
I'm not sure why we're calling apply here, it's probably per your need, but you could expand?

Reply:
Because I don't know exact number of arguments that are in `Args`